src/infima.css

```diff
@@ -1,196 +1,204 @@
 :root {
```

Comment:
These changes need to be simplified. GitHub is showing 6000+ lines have been changed, which is too much to be reviewed.

Reply:
I only added a few lines, maybe Prettier formated the code diferently? Should I use standard settings on vscode or is there other solution to avoid problems in the future?